tests/data/first_fails/expected_results.json

```diff
@@ -1,6 +1,10 @@
 {
   "version": 2,
   "status": "fail",
+  "test-environment": {
+    "bash": "5.0.17(1)-release",
+    "bats": "Bats 1.5.0"
+  },
```

Comment:
Due to changes in the `tests.bats` file, this does not _need_ to be in the expected_results. I thought I'd add at least one to show what the actual results will look like.